docs/transparent-validation/sd-wire/usage-validation.md

```diff
@@ -116,15 +151,15 @@ Dev: 0, Manufacturer: SRPOL, Serial: sd-wire_11, Description: sd-wire
 
 Using SDWire there is no need to disconnect SD card from DUT.
 
-### Scenario using SDWire and RTE
+## Scenario when SDWire and RTE are in use
 
 1. Insert SD card to the SDWire.
-1. Insert SDWire into the DUT and connect it to the TS with micro-USB-->USB
+2. Insert SDWire into the DUT and connect it to the TS with micro-USB-->USB
```

Comment:
@sulewskiprzemyslaw there is no need to change the numeration from 1, 1, 1 to 1,2,3 in the markdown. It renders correctly and it makes easier to change the steps if needed.

Reply:
OK, I understand